test_scripts/Capabilities/PersistingHMICapabilities/019_RAI_use_default_capability_does_not_contain_correspond_cap.lua

```diff
@@ -33,12 +33,10 @@ local requests = {
 
 --[[ Local Functions ]]
 local function updateHMICaps(pMod, pRequest)
-  for key, _ in pairs (hmiDefaultCap) do
-    if key == pMod then
-      hmiDefaultCap[pMod][pRequest] = nil
-      if not pMod == "Buttons" then
-        hmiDefaultCap[pMod].IsReady.params.available = true
-      end
+  hmiDefaultCap[pMod][pRequest] = nil
```

Comment:
@VjKlepikov @aderiabin Do we really need changes in the existing atf test scripts? Because the commit message says we are only adding new ones. Maybe we can create a separate commit for it

Reply:
@LitvinenkoIra Yes we need this change. The change is located in separate commit.